docs/git/cli/lesson.md

```diff
@@ -0,0 +1,81 @@
+---
+id: lesson
+title: Lesson
+sidebar_label: Lesson
+---
+
+import Feedback from "@theme/Feedback";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Learning Objectives
+
+By the end of this class, you should be able to:
+
+- To _clone_ a repository using Git CLI
+- To _commit_, _fork_ a repository, and _push_ changes using Git CLI
```

Comment:
They won't really be forking from the CLI, they'll still be forking through the web UI, but then more be setting up their local clone to use their fork... I'm not sure of a concise way to express that, though...